src/scss/_brand.scss

```diff
@@ -0,0 +1,141 @@
+////
+/// @group o-label
+/// @link http://registry.origami.ft.com/components/o-label
+////
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oLabelGet($variables, $from: null) {
+	@return oBrandGet($component: 'o-label', $variables: $variables, $from: $from);
+}
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oLabelSupports($variant) {
+	@return oBrandSupportsVariant($component: 'o-label', $variant: $variant);
+}
+
+// Define shared brand configurations
+$_label-border-width: 1px;
+$_o-label-shared-brand-config: (
+	default-font-size: nth(oTypographyGetScale(0), 1) * 1px,
```

Comment:
Remember we discussed this and I suggested it would be better to keep brand variables as css properties? That might have been a lie. I'm thinking it would be clearer if this was a scale value and we used o-typography in a more typical way. What do you think?

`oTypographySans($scale: _oLabelGet('font-scale'), $line-height: 1em)`. 

Plus this means progressive font loading is handled properly.